[#725] Added hover text on links in news page

ameybhavsar24 opened the pull request:
Shows small text when hovered on links to describe the action.
## Description
<!--- Describe in details the proposed mods -->
As already mentioned in #725 , this PR tackles:
* Adds hover text for links in the news page.

## Checklist
- [x] Ready for review! :rocket:

## Fixes
<!-- Please insert the issue numbers after the # symbol -->
- Fixes #725

bfabio reviewed (changes requested):
Please note the message should be localized Italian and English. You can take a look at `_data/l10n.yml` and use `site.data.l10n[active_lang].t.your_new_message_key` in the templating engine.

The Italian message for "Share on Twitter" is "Condividi su Twitter".

ameybhavsar24 commented:
Ok, I'll see that.

ameybhavsar24 commented:
I have successfully localised the message.
Resolving merge conflicts now.

ameybhavsar24 commented:
I have resolved the conflicts now. @bfabio Can you review?

bfabio reviewed (approved):
Great :+1:

sebbalex commented on _data/l10n.yml:
```diff
@@ -77,7 +77,9 @@ it:
     home_tools_subtitle: Ci appoggiamo a strumenti già esistenti e molto conosciuti per fornirti le informazioni, il codice e il supporto di cui hai bisogno
-
+    
+    postarticle_socials_icon_title_prefix: Condividi su
```
maybe there is no need to create this, we could do as we did here

https://github.com/italia/developers.italia.it/blob/master/_includes/share.html#L3

bfabio replied:
@sebbalex Nice catch!

ameybhavsar24 replied:
@sebbalex I guess something like
`{{ t.share_msg | capitalize }} {{ t.on_ }} {{ social[0] | capitalize }}`
should render to `Share on Twitter`.
I'll make the changes.

sebbalex commented on _data/l10n.yml:
```diff
@@ -311,7 +313,9 @@ en:
     home_tools_subtitle: We use existing and well-known tools to give you all information, source code and support that you need
-
+    
+    postarticle_socials_icon_title_prefix: Share on
```
same as above

ameybhavsar24 commented:
@sebbalex 
Removed `postarticle_socials_icon_title_prefix` key from _data/l10n.yml and changed social title message to use existing keys
All good?

sebbalex reviewed (approved):
> @sebbalex
> Removed `postarticle_socials_icon_title_prefix` key from _data/l10n.yml and changed social title message to use existing keys
> All good?

@ameybhavsar24 of course, it looks good, thanks for your work!
